fedmsg_meta_umb/mbs.py

```diff
@@ -27,7 +27,9 @@ class MBSProcessor(BaseProcessor):
     __link__ = 'https://mbs.engineering.redhat.com'
     __docs__ = 'https://gitlab.cee.redhat.com/rhel-appstream/module-maintainer-guide'
     __obj__ = 'Module Builds'
-    __icon__ = 'https://apps.fedoraproject.org/img/icons/modularity.png'
+    __icon__ = (
+        'https://datagrepper-prod-datanommer.int.open.paas.redhat.com/'
```

Comment:
Is there any way around hard-coding the internal FQDN?

Reply:
Not really (we haven't come up with one).  This is *the* plugin library that produces *the* URLs.